.travis.yml

```diff
@@ -0,0 +1,39 @@
+language: python
+python: '3.5'
+
+# test multiple Django versions
+matrix:
+  include:
+    # Python 3.x envs
+    - env: DJANGO_VERSION=2.2
+    - env: DJANGO_VERSION=1.9
+    - env: DJANGO_VERSION=1.5
+      python: '3.4'  # Python 3.5+ will error on Django < 1.8: https://stackoverflow.com/a/36000103/3431180
+      # Poetry fails to resolve the dependencies, so don't use it
+      before_install: skip
+      install:
+        # django 1.5 needs pytest-django 2.9 needs pytest 3.5 needs pytest-cov 2.6
+        - pip install django==1.5 pytest-django==2.9 pytest==3.5 pytest-cov==2.6
+        - pip install -e .  # won't be able to do this if setup.py is removed!
+    # Python 2.7 envs
+    - env: DJANGO_VERSION=1.11  # latest Django that supports 2.7
+      python: '2.7'
+      # Poetry fails to resolve the dependencies, so don't use it
+      before_install: skip
+      install:
+        # pytest 5+ does not support Python 2.7
+        - pip install django==1.11 "pytest<5" pytest-django pytest-cov
+        - rm pyproject.toml  # editable install reads from here somehow and bugs out on 2.7
+        - pip install -e .  # won't be able to do this if setup.py is removed!
+
+
+before_install:
+  - pip install poetry;
```

Comment:
woops this should be changed back to a one-liner as I moved the if logic that I was doing here into the matrix instead (which is definitely cleaner!)